apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java

```diff
@@ -124,21 +127,32 @@ public Patient read(@IdParam IdType patientId, RequestDetails requestDetails) {
       throw new ResourceNotFoundException(patientId);
     } finally {
       beneByIdQueryNanoSeconds = timerBeneQuery.stop();
+
       TransformerUtils.recordQueryInMdc(
-          String.format("bene_by_id.%s", includeIdentifiersMode.name().toLowerCase()),
+          String.format(
+              "bene_by_id.include_%s",
+              includeIdentifiersValues
+                  .toString()
+                  .replace("[", "")
+                  .replace("]", "")
+                  .replace(",", "")
+                  .replace(" ", "_")),
           beneByIdQueryNanoSeconds,
```

Comment:
This code is kind of awkward since it relies on the string serialization format of a list. Perhaps something like `String.join("_", includeIdentifiersValues)` is clearer.

Reply:
TY! That is much clearer.